app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/MaintenanceView.kt

```diff
@@ -0,0 +1,252 @@
+package com.example.keniphhomemaintenance.maintenance_screen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.focus.FocusDirection
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import com.example.keniphhomemaintenance.R
+
+val maintenanceViewModel = MaintenanceViewModel()
+
+@Composable
+fun MaintenanceScreen() {
+    val maintenanceList = maintenanceViewModel.maintenanceItemList.value
+    val isDialogOpen = remember { mutableStateOf(false) }
+
+    Scaffold(
+        floatingActionButton = {
+            FloatingActionButton(
+                modifier = Modifier.size(dimensionResource(id = R.dimen.add_maintenance_item_button_size)),
+                shape = CircleShape,
+                onClick = {
+                    isDialogOpen.value = true
+                }
+            ) {
+                Icon(Icons.Default.Add, contentDescription = "Button to add a maintenance item")
+            }
+        },
+        floatingActionButtonPosition = FabPosition.Center
+    ){
+            ShowDialog(isDialogOpen)
+    }
+
+    MaintenanceItemsList(maintenanceList)
+
+}
+
+
+
+@Composable
+fun MaintenanceCard(maintenanceItem: MaintenanceItem) {
+    Spacer(modifier = Modifier.width(dimensionResource(R.dimen.maintenance_card_start_padding)))
+    Row(
+        modifier = Modifier
+            .padding(vertical = dimensionResource(R.dimen.maintenance_card_row_vertical_padding))
+            .padding(start = dimensionResource(R.dimen.maintenance_card_row_start_padding))
+            .fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Image(
+            painter = painterResource(R.drawable.ic_baseline_build_24), 
```

Comment:
Image hard coded as generic wrench icon. Will change in the future when different types of maintenance items are chosen by the user.